downloads/release-notes/1.2.7/RELEASE_NOTE.md

```diff
@@ -2,7 +2,7 @@
 layout: release-note
 title: Release note
 ---
-# Overview of jBallerina {{ site.data.stable-latest.metadata.version }}
+## Overview of jBallerina {{ site.data.stable-latest.metadata.version }}
```

Comment:
The version is not replaced. Please check for the other places as well. `{{ site.data.stable-latest.metadata.version }}` should be replaces with `1.2.7`